### ISSUE TITLE
No wmv support?

### ISSUE THREAD

**Issue by EdwinKM** [easymodo/qimgv#233]
Without turning this program in a full fledged movie player. But why is all code deliberately only speaking about "mp4" and "webm". All other movie types are ignored. This will probably just run fine using libav?

**Comment by easymodo** [comment]
No particular reason. It's just those two formats are often used in places where previously was used gif.

**Comment by easymodo** [comment]
Yes it should will run everything mpv can run. I suppose i could expose formats option via settings.conf in case anyone really wants to use this as a videoplayer.. why though

**Comment by deinlandel** [comment]
Mkv is also used for looped videos, please add it to the list

**Comment by danielfomin96** [comment]
Are there any plans to enable this feature?

**Comment by easymodo** [comment]
Added `mpeg mkv wmv avi mov flv` in addition to `webm/mp4`. I think that covers all of the popular ones?

build from master if you need this right away
